mkdocs.yml

```diff
@@ -29,35 +29,37 @@ nav:
                   - C++ Actions API: sc-memory/api/cpp/extended/agents/actions.md
                   - C++ Agent context API: sc-memory/api/cpp/extended/agents/agent_context.md
                   - C++ Modules API: sc-memory/api/cpp/extended/agents/modules.md
-          - C++ Guides:
-              - C++ Simple guide for implementing agent: sc-memory/api/cpp/guides/simple_guide_for_implementing_agent.md
       - Python API:
           - Python Core API: https://github.com/ostis-ai/py-sc-client
           - Python Extended API: https://github.com/ostis-ai/py-sc-kpm
       - TypeScript API:
           - TypeScript Core API: https://github.com/ostis-ai/ts-sc-client
-      - Sc-element types: scs/sc_element_types.md
+      - sc-element types: scs/sc_element_types.md
       - SCs-code: scs/scs.md
-  - Utils:
-      - Agents: sc-kpm/kpm.md
   - Tools:
       - Knowledge Base Builder: sc-tools/sc_builder.md
       - Knowledge Base repo file: sc-tools/kb_repo_file.md
-      - Sc-machine Runner: sc-tools/sc_machine.md
-      - Migrate to new agent api: sc-tools/migrate_to_new_agent_api.md
+      - sc-machine Runner: sc-tools/sc_machine.md
+  - Build instructions:
+      - Quick start: build/quick_start.md
+      - Build system: build/build_system.md
+      - CMake flags: build/cmake_flags.md
+      - Configuration file: build/config.md
+  - Architecture:
+      - Core agents: sc-kpm/kpm.md
```

Comment:
I couldn't find decomposition agent in sc-machine

Reply:
Yes, we removed it here https://github.com/ostis-ai/sc-machine/pull/378